tests/search/test_neighbourhood.py

```diff
@@ -98,6 +98,7 @@ def test_neighbourhood_params_does_not_raise_for_valid_arguments(
         # fmt: on
     ],
 )
+# TODO
```

Comment:
This test fails but I don't know (yet) why.

```
______________________________________________________________________________ test_compute_neighbours[18-20-34-True-False-99-expected_neighbours_check3] ______________________________________________________________________________
...

    )
    def test_compute_neighbours(
        rc208,
        weight_wait_time: int,
        weight_time_warp: int,
        nb_granular: int,
        symmetric_proximity: bool,
        symmetric_neighbours: bool,
        idx_check: int,
        expected_neighbours_check: set[int],
    ):
        """
        Tests ``compute_neighbours`` on several well-understood cases.
        """
        params = NeighbourhoodParams(
            weight_wait_time,
            weight_time_warp,
            nb_granular,
            symmetric_proximity,
            symmetric_neighbours,
        )
        neighbours = compute_neighbours(rc208, params)
    
        assert_equal(len(neighbours), rc208.num_locations)
        assert_equal(len(neighbours[0]), 0)
    
        # We compare sets because the expected data (from the old C++
        # implementation) sorts by client ID (ascending), not proximity.
>       assert_equal(set(neighbours[idx_check]), expected_neighbours_check)
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: {9, 10, 11, 12, 13, 14, 15, 16, 17, 20, 22, 24, 47, 52, 53, 55, 56, 57, 58, 59, 64, 65, 66, 69, 74, 80, 82, 83, 86, 87, 88, 90, 91, 98}
E        DESIRED: {9, 10, 11, 12, 13, 14, 15, 16, 20, 22, 24, 47, 52, 53, 55, 56, 57, 58, 59, 60, 64, 65, 66, 69, 74, 80, 82, 83, 86, 87, 88, 90, 91, 98}
```

Reply:
Turns out that the VRPLIB Solomon instances (from LKH-3) were different in 1) the number of vehicles and 2) the rounding function. I have changed the instances to match the original instances.

Reply:
Can we then also get rid of the new `# TODO` comment? :-)

Reply:
Oops 😄